test-suite/js/src/test/scala/org/scalajs/testsuite/library/ReflectTest.scala

```diff
@@ -307,4 +317,11 @@ object ReflectTest {
   }
 
   trait TraitDisable extends Accessors
+
+  // Regression cases
+
```

Comment:
Link to issue if you classify this as a regression case?

Reply:
It's in the associated test method name, isn't it?

Reply:
Indeed:-/